docs/API/core/app-actions.md

```diff
@@ -48,7 +48,7 @@ To start, use the `Fliplet.Page.onRemoteExecution()` function to register your c
 Fliplet.Page.onRemoteExecution(function (payload) {
   // This code will run when the screen is triggered by an app action.
 
-  console.log('App action sent this payload', payload);
+  // Here you can access the payload sent to the action when it was called. 
```

Comment:
I would prefer keeping the console log as it makes clear the variable can be used.

Reply:
So clients would just copy this snippet and are confused why the console never runs. It does not run on the page it was pasted in and neither does this run when calling the action on demand. Its doesn't seem like there is any way to make this console run. Does that make sense?

Reply:
Ah I see, that makes sense. I'm good with removing the console log then.